network/modules/UDP/socket_wrapper.py

```diff
@@ -0,0 +1,102 @@
+import socket
+
+
+class UdpSocket:
+    """
+    Wrapper for Python's socket module.
+    """
+
+    def __init__(self, socket_instance: socket.socket) -> None:
+        """
+        Parameters
+        ----------
+        instance: socket.socket
+            For initializing Socket with an existing socket object.
+        """
+        if socket_instance is None:
+            self.__socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        else:
+            self.__socket = socket_instance
+
+    def send_to(self, data: bytes, address: tuple) -> bool:
```

Comment:
Instead of using address as a tuple, please send host and port as 2 different parameters, then you can put them into a tuple yourself in the implementation. It's kind of annoying for users to have to do this themselves.

Reply:
Additionally, I would like to move the process of packing the data (see `struct.pact`, and use the `!` or network format, which is big-endian) into the send and recv functions so that users don't have to worry about it.

Reply:
Also, UDP sockets will give an error when data size is too large. Can you setup a while loop to break up the data into smaller chunks, such as 4096 bytes (have this as some sort of constant)